pycbc/inference/models/relbin.py

```diff
@@ -440,20 +481,26 @@ def _loglr(self):
         self._current_wf_parts = {}
         for ifo in self.data:
 
-            det = self.det[ifo]
             freqs = self.fedges[ifo]
             sdat = self.sdat[ifo]
             h00 = self.h00_sparse[ifo]
             end_time = self.end_time[ifo]
             times = self.antenna_time[ifo]
 
             # project waveform to detector frame
-            hp, hc = wfs[ifo]
-            fp, fc = det.antenna_pattern(p["ra"], p["dec"],
-                                         p["polarization"], times)
-            dt = det.time_delay_from_earth_center(p["ra"], p["dec"], times)
-            dtc = p["tc"] + dt - end_time
+            # FIXME: Don't want to do this for LISA!!
+            if self.is_lisa:
+                fp, fc = (1, 0)
+                dt = 0
+                dtc = -end_time
+            else:
+                det = self.det[ifo]
+                fp, fc = det.antenna_pattern(p["ra"], p["dec"],
+                                             p["polarization"], times)
+                dt = det.time_delay_from_earth_center(p["ra"], p["dec"], times)
+                dtc = p["tc"] + dt - end_time
 
+            hp, hc = wfs[ifo]
```

Comment:
Directly add new cython function and set set.lik to that if the detector response is already applied

Reply:
Added new cython function for fd_det_seqeunce waveforms. Changed relbin.py path to use new function.